### PR TITLE
Drop players if they delete their submission message

### PR DESCRIPTION
**Pull request by AlphaKretin**
## Description

Adds `ManualDeckSubmission.message` to the database and implements a `messageDelete` event handler. 

## Checklist

- [x] I am following the [contributing guidelines](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/.github/CONTRIBUTING.md).
- [ ] I have updated any relevant [user guides](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/guides).
- [ ] I have updated any relevant [documentation](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/docs).
- [ ] I have updated the [changelog](https://github.com/DawnbrandBots/emcee-tournament-bot/blob/master/changelog.md), or this change is not user-facing.

### CODE REVIEW

**Comment by kevinlul** [comment]
merge conflict 🤨

**Comment by AlphaKretin** [comment]
> merge conflict 🤨

Both this PR and #462 edit a (different) message in `open.ts`.

File: src/events/messageDelete.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { CommandSupport } from "../Command";
+import { ManualDeckSubmission } from "../database/orm";
+import { dropPlayer } from "../slash/database";
+import { getLogger } from "../util/logger";
+
+const logger = getLogger("messageDelete");
+
+export function makeHandler(bot: Client, support: CommandSupport): (msg: Message | PartialMessage) => Promise<void> {
+	return async function messageDelete(msg: Message | PartialMessage): Promise<void> {
+		support.database.cleanRegistration(msg.channelId, msg.id);
```
Comment (kevinlul) [inline src/events/messageDelete.ts:11]: I believe this needs to be awaited now

**Comment by kevinlul** [comment]
Add a placeholder value in the basic test on the entities, since a new required field was added to ManualDeckSubmission

File: src/slash/open.ts [hunk 2]
```diff
@@ -135,7 +135,7 @@ async function registerParticipant(
 
 	await interaction.update({});
 	await interaction.user.send(
-		`Please upload screenshots of your decklist to ${userAction}.\n**Important**: Please do not delete your message! This can make your decklist invisible to tournament hosts, which they may interpret as cheating.`
+		"Please upload screenshots of your decklist to ${userAction}.\n**Important**: Please do not delete your message! You will be dropped for cheating, as this can make your decklist invisible to hosts."
```
Comment (kevinlul) [inline src/slash/open.ts:138]: bad merge, need backtick quotes

**Review by kevinlul — APPROVED** [review]

**Pull request merged**